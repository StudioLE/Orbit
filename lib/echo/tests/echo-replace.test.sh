#!/bin/bash
set -uo pipefail

# ARGS
ACTUAL_FILE=$1

# START

OUTPUT=$(/srv/lib/echo/tests/echo-replace-test.sh)

echo "${OUTPUT}" > "${ACTUAL_FILE}"
