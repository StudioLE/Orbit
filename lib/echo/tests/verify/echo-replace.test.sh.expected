LINE 1
LINE 2
LINE 3
LINE 4
LINE 5
[2A[0KREPLACE LINE 4
[2BTHIS SHOULD BE LINE 6

LINE 8
LINE 9
LINE 10
[2A[0KREPLACE LINE 9
REPLACE LINE 10
[2B
